lib/OpenCloud/ObjectStore/Resource/AbstractResource.php

```diff
@@ -36,7 +38,7 @@
     /** @var string The FQCN of the metadata object used for the container. */
     protected $metadataClass = 'OpenCloud\\Common\\Metadata';
 
-    /** @var \OpenCloud\Common\Service\ServiceInterface The service object. */
```

Comment:
We can change this specific occurrence back if you prefer (the use statement needs to stay anyways because L53 et al)

However, note that `\OpenCloud\Common\Service\ServiceInterface` doesn't define `getCDNService` which thus would make any decent IDE fail here:

<img width="1221" alt="screen shot 2017-07-18 at 19 31 22" src="https://user-images.githubusercontent.com/878997/28330953-b5de4afa-6bef-11e7-88bc-a22d39694f43.png">